components/colorhandle/index.css

```
@@ -14,6 +14,8 @@
 @import "themes/spectrum-two.css";
 
 .spectrum-ColorHandle {
+	--mod-opacity-checkerboard-position: 50%;
```

```suggestion
    /* @passthrough -- opacity checkerboard customization */
	--mod-opacity-checkerboard-position: 50%;
```

This flag will help the linter know to ignore this passthrough customization.